Parsers/ASimNetworkSession/Parsers/ASimNetworkSessionMicrosoftSysmon.yaml

```diff
@@ -0,0 +1,160 @@
+Parser:
+  Title: Network Session Event ASIM parser for Sysmon (Event 3)
+  Version: '0.0.1'
+  LastUpdated: Nov 11, 2022
+Product:
+  Name: Windows Sysmon
+Normalization:
+  Schema: NetworkSession
+  Version: '0.2.4'
+References:
+- Title: ASIM Network Session Schema
+  Link: https://aka.ms/ASimNetworkSessionDoc
+- Title: ASIM
+  Link: https:/aka.ms/AboutASIM
+Description: |
+  This ASIM parser supports normalizing Sysmon network session events (event 3) collected using the Event or WEF (WindowsEvent table) connectors to the ASIM Process Event normalized schema. 
+ParserName: ASimNetworkSessionMicrosoftSysmon
+EquivalentBuiltInParser: _ASim_NetworkSession_MicrosoftSysmon
+ParserParams:
+  - Name: disabled
+    Type: bool
+    Default: false
+ParserQuery: | 
+let parser = (disabled:bool = false) {
```

Comment:
You need the query to be indented for YAML. Also notice that YAML validation failed. And as long as YAML validation did not work, KQL validation also don't work.